src/apple/macos/ffi.rs

```diff
@@ -0,0 +1,132 @@
+//
+// Sysinfo
+//
+// Copyright (c) 2015 Guillaume Gomez
+//
+
+use core_foundation_sys::array::__CFArray;
+use core_foundation_sys::base::CFAllocatorRef;
+use core_foundation_sys::dictionary::CFMutableDictionaryRef;
+use core_foundation_sys::string::{CFStringEncoding, CFStringRef};
+use core_foundation_sys::url::CFURLRef;
+use libc::{c_char, size_t};
+
+pub(crate) use super::super::ffi::*;
```

Comment:
```suggestion
pub(crate) use crate::ffi::*;
```

Reply:
This doesn't work for me. I think its because of the 2015 module system that `sysinfo` is currently using.

Reply:
Maybe `crate::sys::ffi`? XD

Reply:
Oh right, done.